plugins/http_plugin/http_plugin.cpp

```diff
@@ -287,13 +328,201 @@ namespace eosio {
                error_results results{websocketpp::http::status_code::too_many_requests, "Busy", error_results::error_info()};
                con->set_body( fc::json::to_string( results, fc::time_point::maximum() ));
                con->set_status( websocketpp::http::status_code::too_many_requests );
+               con->send_http_response();
```

Comment:
`send_http_response` can only be called if `defer_http_response` has first been called.

Reply:
It looks like it is safe to call `defer_http_response` more than once, so you call it before calling `send_http_response`